pom.xml

```diff
@@ -2692,7 +2692,7 @@
     <profile>
       <id>scala-2.12</id>
       <properties>
-        <scala.version>2.12.3</scala.version>
+        <scala.version>2.12.4</scala.version>
```

Comment:
This update picks up a fix for another scalac bug that also prevents the 2.12 build from working, but, has no effect on 2.11